onnxruntime/test/onnx/main.cc

```diff
@@ -441,9 +443,77 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"resize_upsample_sizes_nearest", "not implemented yet"},
       {"resize_upsample_sizes_nearest_floor_align_corners", "not implemented yet"},
       {"resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric", "not implemented yet"},
-      {"scatternd", "not implemented yet"},
-      {"yolov3", "regression in resize opset 10 shape inference"},
```

Comment:
"Yolov3" - Should be fixed by a shape inference fix in ONNX and hence removing out of exclusion.